azurelinuxagent/common/protocol/wire.py

```diff
@@ -164,7 +156,7 @@ def get_artifacts_profile(self):
         logger.verbose("Get In-VM Artifacts Profile")
         return self.client.get_artifacts_profile()
 
-    def download_ext_handler_pkg_through_host(self, uri, destination):
+    def _download_ext_handler_pkg_through_host(self, uri, destination):
```

Comment:
made this one private for clarity, since it is not part of the Protocol interface